docker-compose.prod.yml

```diff
@@ -179,8 +179,6 @@ secrets:
 
 volumes:
   mongo-data:
-    name: workbench_mongo_data
-    external: true
```

Comment:
Wieso das rausnehmen? Damit ist die DB nur noch in einem virtuellen volume oder?

Reply:
Hmm, ja, kann sein. Ich weiß nicht genau, was du mir virtuell meinst. Sie ist dann in einem benannten Volume, das vom Portainer gemanaged wird, aber nicht von uns erstellt. (und kann so auch nicht mehr auf einem NFS-Volume liegen, was zu schwer zu debuggenden Fehlern beim Startup geführt hat.)

Reply:
Virtuell ist glaube ich nicht ganz der richtige Begriff, sondern "anonymous". 

Nicht portainer managed das dann, sondern docker. Anstatt dass die Daten dann in einem von uns definierten Verzeichnis abgelegt werden, landet dass dann irgendwo im docker-Verzeichnis (und das wird dann z.B. auch gelöscht wenn du sowas machst wie "docker-compose down -v" ).

Reply:
Wir lassen den Namen und das external Attribute drin. (Hat nichts damit zu tun, ob das Volume lokal oder external ist, wie zuerst von mir gedacht.)